lib/buffer-serializer.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+var configured;
+
+configured = false;
+
+module.exports = (bufferSerializerModule, OtDate) => {
+    if (! configured) {
```

Comment:
jslint likes "!codehere", but we don't use that.

How about we don't care until we implement a JS pretty printer?